package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "functions-quickstart-javascript-azd",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start": "func start",
+    "test": "echo \"No tests yet...\""
+  },
+  "dependencies": {
+    "@azure/functions": "^4.0.0"
```

We're all the way up to 4.5.0 now.  We should update, and run npm install again.  
```suggestion
    "@azure/functions": "^4.5.0"
```

---

left as is as discussed